creator-cards/sean-killeen.json

```diff
@@ -22,12 +22,12 @@
   ],
   "feeds": [
     {
-      "type": "LinkTagAtomType",
+      "type": "personal",
```

Comment:
ℹ️ I think this is correct now but not sure why it was previously `LinkTagAtomType` - leftover from a prior conversation perhaps?

Reply:
Hey @SeanKilleen yea, that is our old format. We changed that some time ago to be either "dotnet" or "personal" and got rid of the older types, using personal is fine :)

If you only have .NET related content on your feed (or can give us a filtered feed) you can use dotnet